MetaMorpheus.sln

```diff
@@ -25,8 +25,8 @@ Global
 		TestAndReleaseLinux|Any CPU = TestAndReleaseLinux|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{57B2A726-CCA9-432E-B74A-2EFA962D1A74}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{57B2A726-CCA9-432E-B74A-2EFA962D1A74}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{57B2A726-CCA9-432E-B74A-2EFA962D1A74}.Debug|Any CPU.ActiveCfg = Debug|x64
```

Comment:
This should probably still be Any CPU unless there is a strong reason to change.

Reply:
changed back.